opencti-platform/opencti-front/tests_e2e/enableReferences/addReportObservable.spec.ts

```diff
@@ -0,0 +1,203 @@
+    const groupsSettingsPage = new GroupsSettingsPage(page);
+    const groupPage = new GroupPage(page);
+    const groupFormPage = new GroupFormPage(page);
+
+    await page.goto('/dashboard/settings/accesses/groups');
+    await expect(groupsSettingsPage.getSettingsPage()).toBeVisible();
+    await groupsSettingsPage.getAddGroupButton().click();
+    await groupFormPage.fillNameInput(noBypassGroupName);
+    await groupFormPage.getCreateButton().click();
+    await expect(groupsSettingsPage.getGroupInList(noBypassGroupName)).toBeVisible();
+    await groupsSettingsPage.getGroupInList(noBypassGroupName).click();
+    await groupPage.getEditButton().click();
+    await groupFormPage.getRolesTab().click();
+    await groupFormPage.getSpecificRuleCheckbox(noBypassRoleName).click();
+    await expect(groupFormPage.getSpecificRuleCheckbox(noBypassRoleName)).toBeChecked();
+  });
+
+  test('Create basic user', async ({ page }) => {
+    const usersSettingsPage = new UsersSettingsPage(page);
+    const userPage = new UserPage(page);
+    const userFormPage = new UserFormPage(page);
+
+    await page.goto('/dashboard/settings/accesses/users');
+    await expect(usersSettingsPage.getSettingsPage()).toBeVisible();
+    await usersSettingsPage.getAddUserButton().click();
+    await userFormPage.fillNameInput(noBypassUserName);
+    await userFormPage.fillEmailInput(noBypassUserLogin);
+    await userFormPage.fillPasswordInput(noBypassUserPassword);
+    await userFormPage.fillPasswordConfirmationInput(noBypassUserPassword);
+    await userFormPage.getCreateButton().click();
+    await expect(usersSettingsPage.getUserInList(noBypassUserName)).toBeVisible();
+    await usersSettingsPage.getUserInList(noBypassUserName).click();
+    await userPage.getEditButton().click();
+    await userFormPage.getGroupsTab().click();
+    await userFormPage.getSpecificGroupCheckbox(noBypassGroupName).click();
+    await expect(userFormPage.getSpecificGroupCheckbox(noBypassGroupName)).toBeChecked();
+    await userFormPage.getSpecificGroupCheckbox('Default (Max Confidence').click();
+    await expect(userFormPage.getSpecificGroupCheckbox('Default (Max Confidence')).not.toBeChecked();
+  });
+});
+
+test.describe('Authenticate no bypass user', () => {
+  test.use({ storageState: { cookies: [], origins: [] } });
+  test('Authenticate basic user', async ({ page }) => {
+    const dashboardPage = new DashboardPage(page);
+    const loginPage = new LoginPage(page);
+    await page.goto('/');
+    await expect(loginPage.getPage()).toBeVisible();
+    await loginPage.fillLoginInput(noBypassUserLogin);
+    await loginPage.fillPasswordInput(noBypassUserPassword);
+    await loginPage.getSignInButton().click();
+    await expect(dashboardPage.getPage()).toBeVisible();
+    await page.context().storageState({ path: noBypassUserAuthFile });
+  });
+});
+
+test('Add and remove observable from Observables tab of a Report as Admin user', async ({ page }) => {
```

Comment:
question: is it intended that this "test" is not inside a test.describe block ?

Reply:
Kind of! The describe blocks I used before were either to: group up tests that are dependant on each other (for the role/group/user creation), or to run the tests in the describe block with a specific context (the authentication describe block is used to run the authentication in a new browser context, to be able to log in the new user without having to log out the main user).
Since this test is independant from any others, I decided to not include it in a describe. 
But the structure of the test file and the usage of describe blocks in general could be up to debate for sure